iothub/device/src/DeviceClient.cs

```diff
@@ -482,14 +482,16 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         public Task SendEventAsync(Message message, CancellationToken cancellationToken) => InternalClient.SendEventAsync(message, cancellationToken);
 
         /// <summary>
-        /// Sends a batch of events to a hub. Requires AMQP or AMQP over WebSockets.
+        /// Sends a batch of events to IoT hub. Use AMQP or HTTPs for a true batch operation. MQTT will just send the messages one after the other.
+        /// For more information on routing <see href="https://docs.microsoft.com/en-us/azure/iot-edge/module-composition?view=iotedge-2018-06#declare-routes"/>
```

Comment:
this routing info wouldn't be relevant for the device client, correct?

Reply:
Yes good catch. Thanks.